backend/src/server.js

```diff
@@ -20,28 +20,30 @@ const driver = getDriver()
 
 const createServer = options => {
   const defaults = {
-    context: async ({ request }) => {
-      const user = await decode(driver, request.headers.authorization)
+    context: async ({ req }) => {
```

Comment:
Can you explain this change?
I see cases where I would like to go through the code base and do the opposite where I spell it  out further, but I respect that others like to shorten variables, but if I lengthen and you shorten, we are kinda working against each other, or should we try to decide a standard and all stick by it?

Reply:
Same, same … 😂

Reply:
Again, not my decision, but `apollo`'s decision. I would spell it `request` and so does `graphql-yoga`, but `apollo` uses `req`.

Reply:
Read [this](https://github.com/prisma/graphql-yoga/issues/449#issuecomment-440595783) for the **why**.

Reply:
Thanks for letting us understand. 👍🏼